arduino-ide-extension/package.json

```
@@ -163,11 +163,7 @@
   ],
   "arduino": {
     "cli": {
-      "version": {
-        "owner": "arduino",
-        "repo": "arduino-cli",
-        "commitish": "71a8576"
-      }
+      "version": "0.32.0-rc.1"
```

```suggestion
      "version": "0.32.0"
```
The production release is now available:
https://github.com/arduino/arduino-cli/releases/tag/0.32.0